### DISCUSSION TITLE
Does this GitHub Action support private PYPI instances?

### DISCUSSION THREAD

**Discussion by cmac4603** [messense/maturin-action#231] [Q&A]
I've searched the issues and discussions in this repo, does this Actions workflow support publishing to private PYPI running servers?

**Comment by messense** [comment]
Probably, you can try set some env vars for upload, namely `MATURIN_REPOSITORY_URL`, `MATURIN_USERNAME` and `MATURIN_PASSWORD`/`MATURIN_PYPI_TOKEN`.

**Comment by cmac4603** [comment]
@messense Thanks! Will give that a go although I don't see those vars anywhere in the code or docs?

**Reply by messense** [reply]
See `maturin upload --help`.

**Comment by cmac4603** [comment]
Oh wait... https://github.com/PyO3/maturin-action/blob/b9e8f88fd4448fdecf5095864cdc7e39a544aa9f/src/index.ts#L613
